### ISSUE TITLE
Automatically move focus to working tree  compare when opening changes from time line view

### ISSUE THREAD

**Issue by jvesouza** [microsoft/vscode#96750]
Issue Type: <b>Feature Request</b>

In the time line view wee can open changes pressing the enter key or pressing the application key and chosing open changes. 
I think that from the point of view of accessibility it would be very interesting if the focus were automatically moved to the working tree compare.

CC @isidorn 

VS Code version: Code - Insiders 1.45.0-insider (a0fe37870c42e0704a492cdc6b8550b4cf23f63c, 2020-04-30T05:44:45.723Z)
OS version: Linux x64 5.6.8-arch1-1

**Comment by isidorn** [comment]
Yeah aboslutely agree. Pressing enter should move focus same as it does in the Explorer.
@eamodio let me know if you need help here.

**Comment by eamodio** [comment]
@isidorn It seems like we need a good pattern here. `enter` for me in the Explorer starts a rename, while in SCM it opens the changes and sets focus. `space` in Explorer opens a file but preserves focus, and doesn't do anything in SCM.

**Comment by isidorn** [comment]
@eamodio the pattern is the following:

Enter should move focus, space does not.

Special case: Mac and explorer. There Enter is the rename (to be aligned with Native Finder). And Cmd + Down is open it and move focus.

So the timeline just do the Enter moves focus, space does not.

fyi @joaomoreno for SCM

**Comment by eamodio** [comment]
This should be fixed in the latest insiders now, see https://github.com/microsoft/vscode/issues/110496

**Comment by isidorn** [comment]
@eamodio thanks for working on this.
@jvesouza will probably try it out soon and provide feedback

**Comment by jvesouza** [comment]
In my opinion it was great.
Thanks.

**Comment by isidorn** [comment]
@jvesouze Thanks for trying it out. Adding verified label.
